projects/gigya/src/cms-components/gigya-js/gigya-js.service.ts

```diff
@@ -0,0 +1,147 @@
+import { Injectable, NgZone } from '@angular/core';
+import {
+  BaseSiteService,
+  LanguageService,
+  ExternalJsFileLoader,
+  WindowRef,
+  GlobalMessageService,
+  AuthRedirectService,
+  GlobalMessageType,
+  User,
+  UserService,
+} from '@spartacus/core';
+import { GigyaConfig } from '../../config/gigya-config';
+import { BehaviorSubject } from 'rxjs';
+import { GigyaAuthService } from '../../auth/facade/gigya-auth.service';
+
+export function gigyaLogin(response) {
```

Comment:
Could we remove the coupling of this `GigyaJSService` with `GigyaRaasComponent`?